Source/RicohAPIAuth.swift

```diff
@@ -83,7 +84,7 @@ public class AuthClient {
             return
         }
         
-        if NSDate().compare(expire!) == NSComparisonResult.OrderedAscending {
+        if NSDate().compare(expire!.dateByAddingTimeInterval(-expireMargin)) == NSComparisonResult.OrderedAscending {
```

Comment:
If `expireMargin` defiled `-10`, above code is to be `dateByAddingTimeInterval(expireMargin)`, I think it is simpler.